src/scripts/templates/gift_card.js

```diff
@@ -1,43 +1,36 @@
+import QRCode from 'qrcode';
+
 /**
  * Gift Card Template Script
  * ------------------------------------------------------------------------------
  * A file that contains scripts highly couple code to the Gift Card template.
  */
 
-import $ from 'jquery';
-
-const config = {
-  qrCode: '#QrCode',
-  printButton: '#PrintGiftCard',
-  giftCardCode: '.giftcard__code',
+const selectors = {
+  qrCodeCanvas: '[data-gift-card-qr]',
+  printButton: '[data-gift-card-print]',
+  giftCardCode: '[data-gift-card-digits]',
 };
 
-// new QRCode($qrCode[0], {
-//   text: $qrCode.attr('data-identifier'),
-//   width: 120,
-//   height: 120,
-// });
+// This is the QR code that allows customers to use at a POS
+document.querySelectorAll(selectors.qrCodeCanvas).forEach((element) => {
+  QRCode.toCanvas(element, element.dataset.identifier);
+});
 
-$(config.printButton).on('click', () => {
-  window.print();
+document.querySelectorAll(selectors.printButton).forEach((element) => {
+  element.addEventListener('click', () => {
+    window.print();
+  });
 });
 
 // Auto-select gift card code on click, based on ID passed to the function
-$(config.giftCardCode).on('click', {element: 'GiftCardDigits'}, selectText);
-
-function selectText(evt) {
-  const text = document.getElementById(evt.data.element);
-  let range = '';
-
-  if (document.body.createTextRange) {
-    range = document.body.createTextRange();
-    range.moveToElementText(text);
-    range.select();
-  } else if (window.getSelection) {
+document.querySelectorAll(selectors.giftCardCode).forEach((element) => {
+  element.addEventListener('click', (evt) => {
+    let range = '';
```

Comment:
No need for this to be `let` anymore
```suggestion
    let range = document.createRange();
```